### ISSUE TITLE
Hard to distinguish brackets

### ISSUE THREAD

**Issue by devbeard** [j0hnm4r5/laser-theme#5]
**Issue:** 
The cursor indicator, the current bracket highlighting and brackets are hard to distinguish (cyan and red). See screenshots below.

With bracket highlights:
![with_bracket_highlight](https://user-images.githubusercontent.com/3913739/43402999-4920c480-9414-11e8-91a0-679d12cf6a43.PNG)

Without:
![without_bracket_highlight](https://user-images.githubusercontent.com/3913739/43403000-493e6760-9414-11e8-8f8c-b5fb591c5429.PNG)

**Suggestion:**
I am not sure if this is possible in vscode, but when highlighting start and end brackets, could the brackets be white in color (conditional styling of some sorts)? Or other form of contrasting cursor/highlight automatically?

**Comment by j0hnm4r5** [comment]
Fixed in 0799e8399b181f354026437e2a3521e09d99d8c1

**Comment by j0hnm4r5** [comment]
I opted to use the lighter blue color, so it's pretty minimally noticable.

My recommendation (what I personally do) is to use [Bracket Pair Colorizer](https://marketplace.visualstudio.com/items?itemName=CoenraadS.bracket-pair-colorizer), with the following config:

```json
        "bracketPairColorizer.consecutivePairColors": [
		"()",
		"[]",
		"{}",
		["#D42450", "#D95702", "#A9D400", "#00A7B5"],
		"#D42450"
	],
	"bracketPairColorizer.highlightActiveScope": true,
	"editor.matchBrackets": false,
	"editor.colorDecorators": false,
```
